### ISSUE TITLE
High vulnerability in dependency (lodash.merge)

### ISSUE THREAD

**Issue by gorner** [gpoitch/ember-cli-less#71]
`yarn audit` output includes the following:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash.merge                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.6.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ [other package]                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ [other package] > ember-cli-less > lodash.merge              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1066                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

The package.json specifies `"lodash.merge": "^3.3.2"` so (assuming the updated version is compatible) a major version bump is required to overcome.

`broccoli-less-single` which itself relies on lodash.merge may also need to be updated (see also https://github.com/gabrielgrant/broccoli-less-single/issues/47).

**Comment by knownasilya** [comment]
@gorner I'll have a look. FYI this is a build time dependency so you shouldn't have any security vulnerabilities in your app now. I'll let you know once it's updated.

**Comment by gorner** [comment]
@knownasilya Understood and agreed, it’s just an annoyance.

**Comment by cythrawll** [comment]
Hi, Can you look into this again?  I know it's just a build dependency, but we have strict adherence guidelines in our CI system.

**Comment by knownasilya** [comment]
Once gabrielgrant/broccoli-less-single#49 is merged I'll update here

**Comment by knownasilya** [comment]
Released as v2.0.3
